go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -188,13 +188,13 @@ func runOvnKube(ctx *cli.Context) error {
 		return fmt.Errorf("failed to initialize exec helper: %v", err)
 	}
 
-	clientset, err := util.NewClientset(&config.Kubernetes)
+	clientset, egressFirewallClientset, err := util.NewClientset(&config.Kubernetes)
```

Comment:
why a specific clientset for egress firewall?

Reply:
@rcarrillocruz there is a seperate clientset for egressFirewall because since it comes from a CRD it's clientset is not created from k8s.io/client-go/kubernetes,.NewForConfig(). The egressFirewall code generator generates it's own clientset in the egressFirewall subdir

Reply:
We need a new clientset because the current clientset uses protobuf and CRDs need to use json. Alex has created an egress ip clientset in his PR. We really dont wan to just keep adding new clientsets per CRD in the OVN code. In the interest of time it is OK to have 2 extra for now, but I asked @alexanderConstantinescu to open this issue so we can consolidate them into a single "ovnkube" clienset in the future:

https://github.com/ovn-org/ovn-kubernetes/issues/1528